backend/inspirehep/records/marshmallow/authors/base.py

```diff
@@ -22,14 +22,21 @@ class AuthorsRawSchema(RecordBaseSchema):
 
 class AuthorsPublicSchema(AuthorsRawSchema):
     class Meta:
-        exclude = FIELDS_TO_EXCLUDE + ["_private_notes", "_collections"]
+        exclude = FIELDS_TO_EXCLUDE + [
+            "_private_notes",
+            "_collections",
+            "acquisition_source",
+        ]
 
     email_addresses = NonHiddenRaw(dump_only=True)
 
 
 class AuthorsPublicListSchema(AuthorsRawSchema):
     class Meta:
-        exclude = AuthorsPublicSchema.Meta.exclude + ["email_addresses"]
+        exclude = AuthorsPublicSchema.Meta.exclude + [
+            "email_addresses",
+            "acquisition_source",
+        ]
 
     acquisition_source = fields.Method("get_acquisition_source")
```

Comment:
Do we still need this (and the method below to remove emails from acquisition source)?

Reply:
You are right. It's not needed anymore. But then why we are actually taking exclude parameter from AuthorsPublicSchema? Maybe we should just inherit from it...

Reply:
It was done because we wanted to hide emails in search but not in the detailed page. But now, since we hide the whole acquisition source I think that we can inherit from AuthorsPublicSchema.

Reply:
Ah, ok I understand the problem here...